.github/workflows/publish-containers.yaml

```diff
@@ -79,6 +79,7 @@ jobs:
       - name: Push helm charts
         run: |
           helm push "flightctl-${{ needs.generate-tags.outputs.helm_tag }}.tgz" oci://${{ env.QUAY_CHARTS }}/
+          helm push "flightctl-${{ needs.generate-tags.outputs.helm_tag }}.tgz" oci://${{ env.QUAY_CHARTS }}/ --version 0-latest
```

Comment:
https://github.com/jumpstarter-dev/jumpstarter-controller/pull/104

That push parameter doesn't seem to exist, where did you see it?